cmd/sync/main.go

```diff
@@ -145,7 +145,12 @@ func main() {
 func createAWSClient(region string) *AWSClient {
 	httpClient := &http.Client{Timeout: connTimeoutInSecs * time.Second}
 	cfg := &aws.Config{Region: aws.String(region), HTTPClient: httpClient}
-	session := session.New(cfg)
+	session, err := session.NewSession(cfg)
```

Comment:
we should propagate this error to the main func and handle it as in https://github.com/nginxinc/nginx-asg-sync/blob/55f3dbabfaee6a4d01a0a1e9cb6f15710ee758c8/cmd/sync/main.go#L56
It makes sense to make this change in a separate commit . seems like a small bug

Reply:
I can make the change here if needed/